### PR TITLE
[Fix #665] Add firefoxVersion JEXL targeting.

### LINKED ISSUE #665: JEXL Targeting: Firefox Version
Waiting on ASR implementation  https://bugzilla.mozilla.org/show_bug.cgi?id=1484207

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
Waiting for `firefoxVersion` to land on Beta to test.

**Comment by glogiotatidis** [comment]
Tested on nightly

File: snippets/base/admin/fields.py [hunk 1]
```diff
@@ -78,6 +81,22 @@ def validate(self, value):
         return value
 
 
+class JEXLFirefoxRangeField(JEXLRangeField):
+    def __init__(self, **kwargs):
+        choices = ([(None, 'No limit')] +
+                   [(x, x) for x in version_list(product_details.firefox_history_major_releases)])
```
Comment (pmac) [inline snippets/base/admin/fields.py]: snippets regularly updates the data here? It won't have recent releases otherwise.

Comment (glogiotatidis) [reply snippets/base/admin/fields.py]: Yes, we do run `update_product_details` in cron. https://github.com/mozmeao/snippets-service/blob/e65443932992180f4ae417715a3c8f0f2b96dda5/scripts/cron.py#L57

**Comment by glogiotatidis** [comment]
Thanks!

**Pull request merged**